packages/eslint-plugin/rules/data-no-store-string-literals.js

```diff
@@ -85,6 +89,123 @@ function collectAllNodesFromObjectPropertyFunctionCalls( context, node ) {
 	return possibleCallExpressionNodes;
 }
 
+function getSuggest( context, callNode ) {
+	return [
+		{
+			desc:
+				'Replace literal with store definition. Import store if neccessary.',
+			fix: ( fixer ) => getFixes( fixer, context, callNode ),
+		},
+	];
+}
+
+function getFixes( fixer, context, callNode ) {
+	const storeName = callNode.arguments[ 0 ].value;
+	const storeDefinitions = {
+		core: {
```

Comment:
Would it be possible to do an optimistic assumption that:
`core` -> `@wordpress/core-data`
`core/(*)` -> `@wordpress/${1}`
variable would be `camelCase( '${1}Store' )`

This way we won't need to keep the list up to date

Reply:
Gotcha! How I missed this 😄  https://github.com/WordPress/gutenberg/pull/28974/commits/5e02dff1e777c8b4e43d1bc95c9f11256e39643a takes care of it